frontend/src/index.js

```diff
@@ -1,12 +1,34 @@
-import React from 'react'
+import React, { Component } from 'react'
 import ReactDOM from 'react-dom'
-import { BrowserRouter as Router, Route } from 'react-router-dom'
-import Dashboard from './components/Dashboard.js'
+import { Provider } from 'react-redux'
+import { Route, withRouter } from 'react-router'
+import { ConnectedRouter } from 'react-router-redux'
+import configureStore, { history } from './redux/configureStore'
+import { Dashboard } from './components'
+
+const store = configureStore()
+
+class ScrollToTop extends Component {
+  componentDidUpdate(prevProps) {
+    window.scrollTo(0, 0)
+  }
+
+  render() {
+    return this.props.children
+  }
+}
+
+const TopScroll = withRouter(ScrollToTop)
```

Comment:
Only include this top scrolling if you think it will definitely be necessary in the future.